addons/mail/static/src/components/chatter/chatter.xml

```diff
@@ -31,13 +31,22 @@
                             threadLocalId="chatter.thread.localId"
                         />
                     </t>
-                    <t t-if="chatter.thread and chatter.hasActivities and chatter.thread.activities.length > 0">
+                    <t t-if="chatter.isActivityBoxVisible">
```

Comment:
You will need to adapt the useStore in the js of the component to add the new keys used in this template (and remove obsolete keys if there are some).